docs/controls/file-pickers/node/index.md

```diff
@@ -0,0 +1,76 @@
+---
+author: kevintcoughlin
+ms.author: keco
+ms.date: 10/02/2018
+ms.topic: overview
+---
+# Microsoft File Browser SDK for Node.js (Preview)
+
+The [Microsoft File Browser SDK](https://www.npmjs.com/package/@microsoft/file-browser) provides a reusable React component for building file experiences with the [Microsoft Graph](https://developer.microsoft.com/en-us/graph). By using the SDK, any React application can render and perform actions on files by providing a valid `access_token`. In this guide, we will show you how to get your React app quickly rendering the File Browser component for Microsoft Graph.
+
+## Setup
+
+### 1. Add Microsoft File Browser as a dependency
+
+In order to use the SDK, you will need to add `@microsoft/file-browser` to your project as a dependency. Using NPM, you can add the dependency by running the following command: 
+
+```shell
+$ npm i --save @microsoft/file-browser
```

Comment:
TIL that was added in `npm@5`. I'd suggest we keep this explicit since a number of folks still use `npm@4`. `npm@5` shipped with Node `v8.x.x` and will completely replace `v6.x.x` in April 2019.

https://github.com/nodejs/Release